src/cli.js

```diff
@@ -11,8 +10,9 @@ program
   .parse(process.argv);
 
 const bundle = JSON.parse(fs.readFileSync(program.bundle, 'utf8'));
-const mainELM = loadELM(bundle);
-logger.info(`exporting file with name: ${mainELM.mainLibrary.library.identifier.id}.json`);
-const moduleJSON = exportModule(mainELM);
-fs.writeFileSync(`${mainELM.mainLibrary.library.identifier.id}.json`, JSON.stringify(moduleJSON));
-logger.info(`${mainELM.mainLibrary.library.identifier.id}.json was exported successfully`);
+const generator = new Generator(bundle, true);
+const ELM = generator.generate();
+
+logger.info(`exporting file with name: ${ELM.name}.json`);
+fs.writeFileSync(`${ELM.name}.json`, generator);
+logger.info(`${ELM.name}.json was exported successfully`);
```

Comment:
```suggestion
const moduleJson = generator.generate();

logger.info(`exporting file with name: ${moduleJson.name}.json`);
fs.writeFileSync(`${moduleJson.name}.json`, JSON.stringify(moduleJson));
logger.info(`${moduleJson.name}.json was exported successfully`);
```

The result of this generate call is module json, not elm. We need to write that to the file, not the generator variable. Otherwise the result will just be a string "[object Object]"